lib/operations/components/idempotency.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+require "operations/components/prechecks"
+
+# Contains logic to handle idempotency checks.
+#
+# Idempotency checks are used to skip operation execution in
```

Comment:
would be nice to add, that it's not just silent failure, but it prevents calling the same event multiple times

Reply:
Isn't this a definition of idempotency? I don't feel like being too redundant in docs :)

Reply:
I'm afraid that it will be used to run operation without raising failure, but actually adding description will not prevent it